modules/400-descheduler/templates/configmap.yaml

```diff
@@ -1,111 +1,119 @@
+{{- range $d := .Values.descheduler.internal.deschedulers }}
+  {{- $deschedulerSpec := $d.spec }}
+  {{- $commonParameters := $deschedulerSpec.deschedulerPolicy.parameters }}
+  {{- $strategies := $deschedulerSpec.deschedulerPolicy.strategies }}
 ---
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: descheduler-policy-configmap
+  name: descheduler-policy-{{ $d.metadata.name }}
```

Comment:
Don't forget that descheduler name should be less than 64 symbols because we use it in another resource name composition.